requirements.txt

```diff
@@ -0,0 +1,5 @@
+ConfigObj
+files
```

Comment:
Do we need to add this to requirements? "files" refers to my files.py. I can rename that file to help clarify that (to "read_file.py" or something like that?).

Reply:
'files' is a library that can be installed, so a rename of the module might be in order.